orchestra/project_api/views.py

```diff
@@ -154,3 +160,35 @@ def message_project_team(request):
         ).format(project)
         raise BadRequest(error_message)
     return {'success': True}
+
+
+class TodoListViewset(ModelViewSet):
+    serializer_class = BulkTodoSerializer
+    permission_classes = (IsSignedUser,)
+    authentication_classes = (OrchestraProjectAPIAuthentication,)
+    filter_backends = (filters.DjangoFilterBackend,)
+    filterset_fields = ('project', 'step',)
+    queryset = Todo.objects.all()
+
+    def get_serializer(self, *args, **kwargs):
+        if isinstance(kwargs.get('data', {}), list):
+            kwargs['many'] = True
+
+        return super().get_serializer(*args, **kwargs)
+
+    def get_queryset(self):
```

Comment:
you no longer need the

```
if ids:
        # filter by them
```

stuff?

Reply:
This was a very important heads up. Thanks! Brought it back and added a small test case.
For some reason, I had thought the `get_queriset` had been unreasonably clever.